driver.go

```diff
@@ -493,16 +494,17 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 	createOpts.ContainerBasicConfig.Labels = driverConfig.Labels
 
 	// Logging
-	if driverConfig.Logging.Driver == "" || driverConfig.Logging.Driver == LOG_DRIVER_NOMAD {
+	switch {
+	case driverConfig.Logging.Driver == "" || driverConfig.Logging.Driver == LOG_DRIVER_NOMAD:
```

Comment:
```suggestion
	switch driverConfig.Logging.Driver {
	case "", LOG_DRIVER_NOMAD:
```

Maybe? Seems simpler like this.

Reply:
ooh nice catch! i was just bulldozing my way through these >_<